.github/workflows/create-test-wf.yml

```diff
@@ -7,25 +7,36 @@ on:
   release:
     types: [published]
 
-# Cancel if a newer run is started
 concurrency:
   group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
-  cancel-in-progress: true
+  cancel-in-progress: false
```

Comment:
```suggestion
```

Reply:
If I understood correctly, this piece of code cancels the run if a new one starts, eg. we push a new commit. I would keep it set to true.